crates/compiler/test_syntax/tests/snapshots/pass/pattern_comma_newlines.expr.formatted.roc

```diff
@@ -1,3 +1,5 @@
-1(i, p) #
- : f
+1(
+    i,
+    p, #
```

Comment:
This seems like it's formatted incorrectly to me. Either we should move the comments above the `1` line, or keep it after the closing parens.

Reply:
I have to guess that this is consistent with what we do for all collections (lists, tuples, records)...

Reply:
If that's the case, we can make a GitHub issue and try to fix that at some point

Reply:
I'm okay with ignoring for now if we do something like that in order to avoid leaving this in

Reply:
https://github.com/roc-lang/roc/issues/7486

Reply:
Looking back at this, I don't see how that is not the right formatting, here's the original:

```ruby
1(i,p#
):f
n
```

and it is being formatted as:

```ruby
1(
    i,
    p, #
) : f
n

```

To me, the comment is obviously meant for the last arg `p` and since the comment there forces this to be multi-line, that causes the args to be outdented.  This feels like the correct formatting to me.